lib/mongo/collection/view/change_stream.rb

```diff
@@ -77,22 +77,33 @@ class ChangeStream < Aggregation
         #   on new documents to satisfy a change stream query.
         # @option options [ Integer ] :batch_size The number of documents to return per batch.
         # @option options [ BSON::Document, Hash ] :collation The collation to use.
-        # @option options [ BSON::Timestamp ] :start_at_cluster_time Only return changes that occurred
-        #   after the specified timestamp. Any command run against the server will return a cluster time
-        #   that can be used here. Only valid in server versions 4.0+.
+        # @option options [ BSON::Timestamp ] :start_at_operation_time Only
+        #   return changes that occurred after the specified timestamp. Any
+        #   command run against the server will return a cluster time that can
+        #   be used here. Only recognized by server versions 4.0+.
         #
         # @since 2.5.0
         def initialize(view, pipeline, changes_for, options = {})
+          @resuming = false
```

Comment:
This distinguishes initial aggregation from resumes.

Reply:
I think that this would work (and be less confusing) if we just didn't initialize `@resume` until the end, since `#resuming?` will just return nil until then.

Reply:
This only hides the fact that while the constructor is executing `@resume` must be set to a specific value. If anything the double assignment makes it clearer that something is going on that potentially needs to be paid attention to.